androbd/src/main/AndroidManifest.xml

```diff
@@ -61,6 +61,15 @@
             android:name=".PidCustomization"
             android:label="@string/customize_display"
             android:configChanges="touchscreen|orientation|screenSize" />
+        <provider
```

Comment:
I am afraid implementation of androidx package kills backwards compatibility ...